examples/timestamping/backend/src/api.rs

```diff
@@ -45,7 +45,8 @@ pub struct TimestampProof {
     /// Actual state hashes of the timestamping service with their proofs.
     pub state_proof: MapProof<IndexCoordinates, Hash>,
     /// Actual state of the timestamping database with proofs.
-    pub timestamp_proof: MapProof<Hash, TimestampEntry>,
+    //TODO: revert change to Raw after refactoring
```

Comment:
What's wrong with the current form of the proof?

Reply:
Nothing is wrong, forgot to remove this todo.